gendarme-ignore.txt

```diff
@@ -294,6 +294,17 @@ T: Java.Interop.JniEnvironmentInfo
 T: Java.Interop.JniFieldInfo
 T: Java.Interop.JniMethodInfo
 T: Java.Interop.JniType
+T: Java.Interop.JniArrayElements
```

Comment:
Could we please alphabetically sort these new entries in with the existing entries? Most (all?) entries are sorted.

Reply:
Fixed